samples/msgext-ai-sentiment-analysis/nodejs/Readme.md

```diff
@@ -0,0 +1,124 @@
+---
+page_type: sample
+description: This sample shows a feature where user can check sentiment for teams message by using messaging extension.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "08/07/2023 04:00:00 PM"
+urlFragment: officedev-microsoft-teams-samples-msgext-ai-sentiment-analysis-nodejs
+---
+
+# Sentiment Analysis with messaging extension
+
+This sample shows a feature where user can check sentiment for teams message by using messaging extension.
+
+## Included Features
+* Bots
+* Open AI
+
+## Interaction with app
+
+![Sentiment Analysis](Images/Sentiment_Analysis.gif)
+
+## Try it yourself - experience the App in your Microsoft Teams client
+Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app package (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).
+
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account (not a guest account)
+-  [NodeJS](https://nodejs.org/en/)
+-  [ngrok](https://ngrok.com/) or equivalent tunneling solution
+-  [M365 developer account](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) or access to a Teams account with the 
+   appropriate permissions to install an app.
```

Comment:
**Content alignment is wrong.
Also, where did you mention about Azure Open AI Key and other details?

Kindly refer my enterprise search app files.**